Update pagination.rst (#7772)
cakephp/docs: en/controllers/pagination.rst

        }
    }

> When the page is out of range, the query will be empty. Other possible solution is to check if there is an empty query
> and the selected page is greater than 1, unset the page and redirect to the first page::

Reviewer: ```suggestion
When an out of range page is requested, the query will be empty. You can check if the page query
is empty and the selected page is greater than 1, and unset the page and redirect to the first page::
```
